modules/apps/frontend-js/frontend-js-metal-web/package.json

```diff
@@ -1,7 +1,6 @@
 {
 	"dependencies": {
 		"metal": "2.16.8",
-		"metal-affix": "2.0.0",
```

Comment:
See comment about this at https://github.com/liferay-frontend/liferay-portal/pull/675#pullrequestreview-565206328.

Mostly irrelevant here since I don't think anyone else was using `metal-affix` ever, but just for completeness.